etc/about.hbs

```diff
@@ -0,0 +1,83 @@
+<html>
+
+<head>
+    <style>
+        @media (prefers-color-scheme: dark) {
+            body {
+                background: #333;
+                color: white;
+            }
+            a {
+                color: skyblue;
+            }
+        }
+        .container {
+            font-family: sans-serif;
+            max-width: 800px;
+            margin: 0 auto;
+        }
+        .intro {
+            text-align: center;
+        }
+        .licenses-list {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
+        .license-used-by {
+            margin-top: -10px;
+        }
+        .license-text {
+            max-height: 200px;
+            overflow-y: scroll;
+            white-space: pre-wrap;
+        }
+    </style>
+</head>
+
+<body>
+    <main class="container">
+        <div class="intro">
+            <h1>Third Party Licenses</h1>
```

Comment:
For reference the rust project copyright template [for the toolchain](https://github.com/rust-lang/rust/blob/main/src/tools/generate-copyright/templates/COPYRIGHT.html) and for [the standard library](https://github.com/rust-lang/rust/blob/main/src/tools/generate-copyright/templates/COPYRIGHT-library.html). 

I think it would also make sense to talk about copyright here, and make the distinction between in-tree code and third-party code. IMHO we could probably mostly take the text of their license template and adapt it to MPL-2.0. 
For the third-party code section here, we could just talk about the third-party licenses.

Would you want to take a look at that, or should i push my suggestions?

Reply:
I believe everything in-tree is MPL-2.0 right? I mostly based the current version of off about:license from firefox.